integrations/jira-example-integration.yml

```diff
@@ -0,0 +1,905 @@
+                    });
+                  }
+                }
+                // Return the JSON payload we want
+                return { data: result };
+              };
+        name: Create Jira Issue payload
+      - action:
+          key: createIssue
+          component:
+            isPublic: true
+            key: atlassian-jira
+            version: LATEST
+        description: ''
+        inputs:
+          ADFdescription:
+            type: value
+            value: ''
+          assignee:
+            type: value
+            value: ''
+          debug:
+            type: value
+            value: 'false'
+          description:
+            type: reference
+            value: createJiraIssuePayload.results.2.value
+          dueDate:
+            type: value
+            value: ''
+          dynamicValues:
+            type: value
+            value: ''
+          fieldValues:
+            type: complex
+            value: []
+          fixVersions:
+            type: value
+            value: ''
+          issueType:
+            type: reference
+            value: createJiraIssuePayload.results.5.value
+          issueTypeId:
+            type: value
+            value: ''
+          jiraConnection:
+            type: configVar
+            value: Jira Connection
+          labels:
+            type: complex
+            value: []
+          priority:
+            type: value
+            value: ''
+          projectId:
+            type: reference
+            value: createJiraIssuePayload.results.4.value
+          reporter:
+            type: value
+            value: ''
+          summary:
+            type: reference
+            value: createJiraIssuePayload.results.1.value
+          versions:
+            type: value
+            value: ''
+        name: Create Issue
+labels:
+  - project-management
+name: Jira Example Integration
+requiredConfigVars:
+  - connection:
+      key: oauth2
+      component:
+        isPublic: true
+        key: atlassian-jira
+        version: LATEST
+    dataType: connection
+    inputs:
+      apiSiteOverride:
+        meta:
+          orgOnly: true
+          visibleToCustomerDeployer: false
+          visibleToOrgDeployer: false
+        type: value
+        value: 'https://example.atlassian.net'
+      authorizeUrl:
+        meta:
+          orgOnly: true
+          visibleToCustomerDeployer: false
+          visibleToOrgDeployer: false
+        type: value
+        value: >-
+          https://auth.atlassian.com/authorize?audience=api.atlassian.com&prompt=consent
+      clientId:
+        meta:
+          orgOnly: true
+          visibleToCustomerDeployer: false
+          visibleToOrgDeployer: false
+        type: value
+        value: clientID
+      clientSecret:
+        meta:
+          orgOnly: true
+          visibleToCustomerDeployer: false
+          visibleToOrgDeployer: false
+        type: value
+        value: clientSecret
```

Comment:
I would maybe leave these blank, so it's clear to someone importing this integration that they need to fill it in.